Sources/BraintreeThreeDSecure/BTThreeDSecureRequest.m

```diff
@@ -99,8 +98,6 @@ - (NSString *)shippingMethodAsString {
 
 - (NSString *)versionRequestedAsString {
     switch (self.versionRequested) {
```

Comment:
We can remove the `versionRequested` enum entirely now, and update the docstrings for the entry-point function that it only does 3DS 2